Updated Models section

atteggiani opened the pull request:
Fixes #774
Fixes #989
Fixes #1012

atteggiani committed:
Updated Models section

atteggiani committed:
Turned some cards to horizontal layout instead of vertical;
Moved the responsiveness CSS to its own file to keep it separated from the general access-nri.css;
Minor changes to styles

atteggiani committed:
Fix link

atteggiani committed:
Moved config docs at the beginning of models information

chrisb13 commented:
As discussed at today's webobs meeting the top "button" link here:
https://docs.access-hive.org.au/pr-previews/1025/models/

Is broken.

atteggiani commented:
> As discussed at today's webobs meeting the top "button" link here: https://docs.access-hive.org.au/pr-previews/1025/models/
> 
> Is broken.

I thought I fixed it but I misspelled `access_models` to `access-models` 🙈

atteggiani committed:
fixup! Fix link

atteggiani committed:
Added config docs to ACCESS-rAM3

atteggiani committed:
Added descriptions to cards

atteggiani commented:
This is a list of the modified URLs, relative to the Hive Docs root URL (https://docs.access-hive.org.au), formatted as:
`<old_url> -> <new_url>`:

- Added `/models/access_models`
- `/models/access-cm -> /models/access_models/access-cm`
- `/models/access-esm	-> /models/access_models/access-esm`
- `/models/access-om	-> /models/access_models/access-om`
- `/models/access-ram	-> /models/access_models/access-ram`
- Added `/models/build_a_model`
- `/models/run-a-model/build_a_model -> /models/build_a_model/build_source_code`
- `/models/run-a-model/create-a-prerelease -> /models/build_a_model/create_a_prerelease`
- `/models/run-a-model -> /models/run_a_model`
- `/models/run-a-model/run-access-cm2 -> /models/run_a_model/run_access-cm2`
- `/models/run-a-model/run-access-esm -> /models/run_a_model/run_access-esm`
- `/models/run-a-model/run-access-om2 -> /models/run_a_model/run_access-om2`
- `/models/run-a-model/run-access-om3 -> /models/run_a_model/run_access-om3`
- `/models/run-a-model/run-access-ram -> /models/run_a_model/run_access-ram3`

atteggiani committed:
Updated figures

atteggiani committed:
Updated components image. Deleted events plugin

KAUR1984 committed:
:lipstick: Added images for build and run a model

KAUR1984 committed:
:lipstick: Removed the previous build a model image

paigem commented on docs/models/access_models/index.md:
```diff
@@ -0,0 +1,53 @@
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-CM</span>
+            <span>
+                ACCESS-CM is a global climate model that includes the atmosphere, land, ocean and sea-ice components, and produces physical climate simulations.
```
```suggestion
                A global climate model that includes the atmosphere, land, ocean and sea-ice components, and produces physical climate simulations
```

paigem replied:
Because by removing the model name, they are no longer full sentences. 😊

paigem commented on docs/models/access_models/index.md:
```diff
@@ -0,0 +1,53 @@
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-ESM</span>
+            <span>
+                ACCESS-ESM is a global climate model with additional land and ocean biogeochemical components to simulate both the physical climate and global biogeochemical cycles.
```
```suggestion
                A global climate model with additional land and ocean biogeochemical components to simulate both the physical climate and global biogeochemical cycles
```

atteggiani replied:
Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/commit/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

paigem commented on docs/models/access_models/index.md:
```diff
@@ -0,0 +1,53 @@
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-OM</span>
+            <span>
+                ACCESS-OM is a global coupled ocean and sea ice model used to understand both physical and biogeochemical processes in the ocean.
```
```suggestion
                A global coupled ocean and sea ice model used to understand both physical and biogeochemical processes in the ocean
```

atteggiani replied:
Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

paigem commented on docs/models/access_models/index.md:
```diff
@@ -0,0 +1,53 @@
+# ACCESS Models
+
+ACCESS models are computer programs that represent major components of the Earth system (e.g., atmosphere, land surface, ocean, sea ice, etc.) using complex mathematical equations. These components can be connected through a coupler to create ACCESS models, which are used to simulate past or future conditions as well as idealised experiments.
```
This is my attempt to make this information a bit more comprehensive for potential new users. I also think we should make it clear here that these are the ACCESS-NRI ACCESS models - there are other ACCESS models that exist that we don't support/develop in any way.

```suggestion
ACCESS models are computer programs that represent key components of the Earth's climate system, including the atmosphere, oceans, land surface, and sea ice. The models use complex mathematical equations to simulate past, present, and future climate conditions, as well as idealised scenarios. Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs, enabling tailored simulations of various aspects of the climate system. These models are developed through collaboration with international climate modeling efforts, but are specifically developed to run on Australia's high-performance computing (HPC) systems, and so are tailored for researchers based in Australia.

The models below are those supported by ACCESS-NRI.
```

atteggiani replied:
I agree we should make it clear that the models are those ACCESS models are those that ACCESS-NRI supports. 
A general considerations on your suggestion is that I would avoid making this introduction too long, as it might deter people from reading it. Below I suggest some modifications to your suggestion to try and improve conciseness and clarity:


> ... to simulate past, present, and future climate conditions, ...

Saying "climate" here is imprecise as Regional models (e.g., ACCESS-rAM3) are often used for weather conditions as well. 
Should we say "weather and climate" instead?

> Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs, enabling tailored simulations of various aspects of the climate system.

I would shorten this bit by removing the last part:
"""
Different ACCESS models are configured with varying combinations of Earth system components to address specific research needs.
"""

> These models are developed through collaboration with international climate modeling efforts, but are specifically developed to run on Australia's high-performance computing (HPC) systems, and so are tailored for researchers based in Australia.
> The models below are those supported by ACCESS-NRI.

I suggest a more concise version:
"""
Developed in collaboration with international climate modeling institutions, these models are optimized for Australia’s high-performance computing (HPC) systems and tailored for Australian researchers. 

Below is a list of ACCESS models supported by ACCESS-NRI.
"""

paigem commented on docs/models/index.md:
```diff
@@ -1,115 +1,55 @@
+            <span class="bold">Model Components</span>
             <span>
-                ACCESS-ESM is a global climate model with additional land and ocean biogeochemical components to simulate both the physical climate and global biogeochemical cycles.
+                ACCESS model components are computer programs representing different chemical, physical or biological parts of the Earth System. These components are interconnected together to form ACCESS models.
```
Same comment as above. A possible option below:

```suggestion
                Overview of the Earth system components that make up the ACCESS models
```

atteggiani replied:
Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

paigem commented on docs/models/run_a_model/index.md:
```diff
@@ -1,11 +1,11 @@
 # Run a Model
 If you are a new user of ACCESS climate models, please refer to how to [Set Up your NCI Account](/getting_started/set_up_nci_account).
```
```suggestion
These instructions are for running the ACCESS models on the Australian National Computational Infrastructure (NCI). If you do not yet have an NCI account, please refer to how to [Set Up your NCI Account](/getting_started/set_up_nci_account).
```

paigem reviewed (changes requested):
Thanks for taking on this large PR @atteggiani! 👏 For the most part, it looks great. I tested a few links, but certainly not all of them. I made some suggested wording updates that I hope will make the text a bit more clear, particularly for new users of the site, and so I am requesting changes before approval.

atteggiani commented on docs/models/index.md:
```diff
@@ -1,115 +1,55 @@
+            <span class="bold" >ACCESS Models</span>
             <span>
-                ACCESS-CM is a global climate model that includes the atmosphere, land, ocean and sea-ice components, and produces physical climate simulations.
+                ACCESS models are computer programs which can be used to simulate past or future conditions of the Earth System, as well as carrying out idealised experiments.
```
Good point!

atteggiani committed:
incorporated Paige's suggestions

atteggiani commented on docs/models/access_models/index.md:
```diff
@@ -0,0 +1,53 @@
+        <div class="card-text-container">
+            <span class="bold" >ACCESS-rAM</span>
+            <span>
+                ACCESS-rAM is an implementation of the UKMO Regional Nesting Suite that allows high-resolution regional atmosphere modelling.
```
Changed in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/9ee788b73b68a233a2cd9de08d9c3d5b2ba79852

chrisb13 commented:
Thanks both.

I think a warning on CM2 is a good idea.

> I think the most important point to get across for users in these model overview and run-a-model pages are the models we _support_ and not distinguishing that from the models we have _released_.

I think this is a good compromise. I would just caution [that this page](https://docs.access-hive.org.au/pr-previews/1025/about/releases/#release-stages-for-model-configurations) has 10 usages of the word "support" so pretty easy for a user to conflate the two ideas. 

Perhaps the page that @paigem (see what I did there?) could strongly tie to the [release](https://docs.access-hive.org.au/pr-previews/1025/about/releases/#release-stages-for-model-configurations)/[support pages](https://docs.access-hive.org.au/pr-previews/1025/about/user_support/)?

I also think if this page is being updated, maybe it's a good time to ensure consistency with these pages:
 1. https://www.access-nri.org.au/models/access-cm/
 2. https://www.access-nri.org.au/models/earth-system-model-esm/
 3. https://www.access-nri.org.au/models/access-om/
 4. https://www.access-nri.org.au/models/access-ram/

I note the OM page has no mention of OM3. Is there any way these two pages could just be repeated or syndicated and that way they don't fall out of sync? Similarly, can https://www.access-nri.org.au/models/* these pages include a link to [the released models](https://forum.access-hive.org.au/search?q=%23access-nri-releases%20tags%3Amodel%20order%3Alatest)? (Ping @NBateman-bit @Corey? GH?)

FYI, there's also some upcoming updates:
 - https://docs.access-hive.org.au/pr-previews/1025/models/access_models/access-esm/ will need to include esm1.6
 - ISSM I guess will be coming soon?
 - ESM3 eventually

paigem commented:
> I would just caution [that this page](https://docs.access-hive.org.au/pr-previews/1025/about/releases/#release-stages-for-model-configurations) has 10 usages of the word "support" so pretty easy for a user to conflate the two ideas.

My reading is that the usage of "support" (as in "user support") on the [model release stages page](https://docs.access-hive.org.au/about/releases/#release-stages-for-model-configurations) is consistent with our usage in the model overview pages. We are indicating that we provide support for the listed models (including ACCESS-CM2).

> I also think if this page is being updated, maybe it's a good time to ensure consistency with these pages:

Could you be more clear with what you mean by "ensure consistency"?

> I note the OM page has no mention of OM3.

I believe this was intended. According to our release stages table, we only promise to share on the main website at a full release.

atteggiani committed:
Incorporate Paige reviews

atteggiani committed:
Incorporate Paige reviews

atteggiani committed:
Incorporate Paige reviews

paigem reviewed (dismissed):
All looks good to me! Thanks @atteggiani! I think this is a big improvement for understandability and clarity of Hive Docs users. 😊  (Note that the PR preview hasn't been updated for some reason.)

atteggiani committed:
Added alert for ACCESS-CM2 support; changed ACCESS-CM to ACCESS-CM2 in How to run a model page.

atteggiani commented:
> But then by that logic why is CM2 there?

I have added an alert in the CM2 pages in https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1025/commits/7c4283a9383c8311f1ec22610617ae81b63e6513.

> Yes but this is a reoccurring issue where it would help if these kinds of changes were more coordinated. Hence, my ping of @NBateman-bit @chanrahan. As you say, perhaps best discussed at a WebOps meeting.

For all the matters related to ACCESS-NRI website, please refer to the WebOps meetings and Zulip channel.
If you want, you can add an item to the [agenda](https://anu365.sharepoint.com/:w:/r/sites/ACCESS-NRI/_layouts/15/doc2.aspx?sourcedoc=%7B6DFCCF70-F604-4A41-8B9F-D54114F7579A%7D&file=Meeting_Agenda_WebOps.docx&action=default&mobileredirect=true) for next WebOps' meeting.

paigem reviewed (approved):
This looks great - the wording is clear and concise. Thanks @atteggiani! 😄

atteggiani commented:
Thank you all for this!
